src/core/integrate.cpp

```diff
@@ -64,6 +65,7 @@
 #include <cstdlib>
 #include <cstring>
 #include <mpi.h>
+#include <Python.h>
```

Comment:
Direct usage of the Python header breaks builds with `-DWITH_PYTHON=OFF`. I don‘t know who needs them, but they are possible. So guards are needed.